File: Counting/3289. The Two Sneaky Numbers of Digitville.cpp
class Solution
{
public:
    vector<int> getSneakyNumbers(vector<int> &nums)
    {
        int n = nums.size();
        vector<int> ans, count(n);
        for (auto val : nums)
        {
            count[val]++;
            if (count[val] == 2)
                ans.push_back(val);
        }
        return ans;
    }
};

File: Counting/3289. The Two Sneaky Numbers of Digitville.py
class Solution:
    def getSneakyNumbers(self, nums: List[int]) -> List[int]:
        ans1, ans2 = -1, -1
        count = Counter(nums)
        for val in count:
            if count[val] == 2:
                if ans1 == -1:
                    ans1 = val
                elif ans2 == -1:
                    ans2 = val
                    break
        return [ans1, ans2]